addons/zylann.hterrain/tools/plugin.gd

```diff
@@ -455,6 +462,20 @@ func _forward_3d_gui_input(p_camera: Camera3D, p_event: InputEvent) -> int:
 
 	var captured_event = false
 	
+	if p_event is InputEventKey:
+		if p_event.keycode == KEY_G and p_event.is_echo() == false and p_event.pressed:
+			captured_event = true
+			_show_brush_size_selector(2, 300, Color.LIGHT_GREEN, _terrain_painter.get_brush_size(), "brush_size_action",
+			func on_value_changed(value):
+				_terrain_painter.set_brush_size(value)\
+			 )
+		elif p_event.keycode == KEY_H and p_event.is_echo() == false and p_event.pressed:
+			captured_event = true
+			_show_brush_size_selector(0, 100, Color.LIGHT_GREEN, _terrain_painter.get_opacity()*100, "brush_opacity_action",
```

Comment:
Those are the min/max values allowed, based on the slider's min/max.
The 2, 300 above was based on the brush size slider that is 2-500. But 500 is a bit too big for the overlay as it is now.
And the 0-100 is the 0 to 100 opacity value.

I found the `MAX_SIZE_FOR_SLIDERS` used in the brush. 
But I can't seen to find where it gets the min_value of 2 from.
It seems the 2 min_value is directly set in the slider node inside the scene of `brush_editor.tscn` and `brush_settings_dialog.tscn`, individually.

And the opacity slider don't have any references either, they are just the default values of the slider, which happens to be 0-100.

I can create those constants in Brush and make the other brush dialogs use them too instead of setting it from the inspector.